ffi/capi/js/package/Makefile

```diff
@@ -2,33 +2,22 @@
 # called LICENSE at the top level of the ICU4X source tree
 # (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-.DEFAULT_GOAL := build
+.DEFAULT_GOAL := target/wasm32-unknown-unknown/release/icu_capi_cdylib.wasm
 FORCE:
 
-ALL_HEADERS := $(wildcard ../../include/*)
 ICU4X_NIGHTLY_TOOLCHAIN ?= "nightly-2022-12-26"
 
-$(ALL_HEADERS):
-
 # 100 KiB, working around a bug in older rustc
 # https://github.com/unicode-org/icu4x/issues/2753
 WASM_STACK_SIZE := 100000
-
-lib: $(ALL_HEADERS)
-	rm -rf lib;
-	mkdir lib;
-	cp ../../include/* lib/
  
-crate/target/wasm32-unknown-unknown/release/crate.wasm: FORCE
+target/wasm32-unknown-unknown/release/icu_capi_cdylib.wasm: FORCE
```

Comment:
Why are you deleting `crate`?

Reply:
It's not an example anymore, so I think it's fine to build it with `-p icu_capi_cdylib`

Reply:
I'm not a fan of `CARGO_TARGET_DIR` because `cargo clean` doesn't know about it

Reply:
You should probably be using `git clean` anyway? It also resets all the other artifacts that are taking up space or could be corrupted, like `node_modules`.